libraries/AP_NavEKF3/AP_NavEKF3_Control.cpp

```diff
@@ -561,7 +577,7 @@ void  NavEKF3_core::updateFilterStatus(void)
     filterStatus.flags.takeoff = expectGndEffectTakeoff; // The EKF has been told to expect takeoff and is in a ground effect mitigation mode
     filterStatus.flags.touchdown = expectGndEffectTouchdown; // The EKF has been told to detect touchdown and is in a ground effect mitigation mode
     filterStatus.flags.using_gps = ((imuSampleTime_ms - lastPosPassTime_ms) < 4000) && (PV_AidingMode == AID_ABSOLUTE);
-    filterStatus.flags.gps_glitching = !gpsAccuracyGood && (PV_AidingMode == AID_ABSOLUTE) && (frontend->_fusionModeGPS != 3); // GPS glitching is affecting navigation accuracy
```

Comment:
Should we really have lost the clause about `_fusionModeGPS`?

Reply:
If PV_AidingMode == AID_ABSOLUTE and we are not using external nav data, then we must be using GPS (there are only two sources of absolute position information, GPS and external nav).